src/pages/Home.js

```diff
@@ -96,7 +102,38 @@ const HomeContent = ({ carePlan }) => {
   const [tab, setTab] = useState(tabs[0]);
 
   if (!carePlan) {
-    return null;
+    const NoCarePlanCard = (text, textColor, left, top) => (
```

Comment:
Can we make this a JSX component
```suggestion
    const NoCarePlanCard = ({text, textColor, left, top}) => (
```
and then render with those params as props

Reply:
Updated: [fbba127](https://github.com/capable-health/capable-care-demo/pull/31/commits/fbba127422c22f4e6131233f090165e99b713363)